moveit_ros/moveit_servo/src/teleop_demo/joystick_servo_example.cpp

```diff
@@ -49,8 +49,8 @@
 
 // We'll just set up parameters here
 const std::string JOY_TOPIC = "/joy";
-const std::string TWIST_TOPIC = "/servo_server/delta_twist_cmds";
-const std::string JOINT_TOPIC = "/servo_server/delta_joint_cmds";
+const std::string TWIST_TOPIC = "/servo_component/delta_twist_cmds";
```

Comment:
Should we just use the `/servo/*` namespace?

Reply:
agree that probably makes sense or /moveit_servo ?

Reply:
:+1: to /servo/*